containers/R_Python/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM clipper/py-rpc:latest
+
+## Set a default user. Available via runtime flag `--user docker` 
```

Comment:
Change the directory containing this file to `containers/R` and change the name of the docker file to `RPythonDockerfile`